prowler

```diff
@@ -175,6 +179,13 @@ if __name__ == "__main__":
         default=None,
         help="Same as -B but do not use the assumed role credentials to put objects to the bucket, instead uses the initial credentials.",
     )
+    parser.add_argument(
+        "-N",
+        "--shodan",
+        nargs="?",
+        default=None,
+        help="Shodan API key used by check ec2_elastic_ip_shodan.",
+    )
```

Comment:
As I said above, for the check's input parameters we should only use the new configuration file.

Reply:
I think that we have to have the same functionality as version 2, at least at the beginning. That's why the best option for keeping both feature is to do it this way.